csrc/python_frontend/python_bindings.cpp

```diff
@@ -361,7 +361,7 @@ void initNvFuserPythonBindings(PyObject* module) {
 
             for (size_t i = 0; i < symbolic_sizes.size(); ++i) {
               TORCH_CHECK(
-                  symbolic_sizes[i] == -1 || symbolic_sizes[i] == 1,
+                  symbolic_sizes[i] > -1,
```

Comment:
What's going on here? Should we no longer define_tensor with symbolic sizes of -1?

Reply:
I thought we are augmenting this, so it should be `>= -1`?

Reply:
Oh maybe I missed this. What does a symbolic size other than -1 or 1 mean?

Reply:
quoting Kevin's commit message:

> 3. Augments the define_tensor operation of the FusionDefinition to allow for the tensor dimensions to be defined as concrete sizes in addition to -1 (symbolic) and broadcast in order to test reduction_factor without having an API for parallelization.

Reply:
I guess that means a static size, maybe we would want to rename the variable.

Reply:
Ugh, I screwed up this statement up.  It should have been `>= -1`.   This also needs comment.  This seemed perfectly clear to me what I was doing but I can see how it is now.

Reply:
Fixed!